src/main/java/net/openhft/chronicle/bytes/internal/BytesInternal.java

```diff
@@ -2141,41 +2141,8 @@ public static void append(@NotNull StreamingDataOutput out, double d)
             out.rawWriteByte((byte) '0');
     }
 
-    private static double asDouble(long value, int exp, boolean negative, int deci) {
-        // these numbers were determined empirically.
-        int leading =
-                Long.numberOfLeadingZeros(value) - 1;
-        if (leading > 9)
-            leading = (27 + leading) >>> 2;
-
-        int scale2 = 0;
-        if (leading > 0) {
-            scale2 = leading;
-            value <<= scale2;
-        }
-        double d;
-        if (deci > 0) {
-            if (deci < 28) {
-                long fives = Maths.fives(deci);
-                long whole = value / fives;
-                long rem = value % fives;
-                d = whole + (double) rem / fives;
-            } else {
-                d = value / Math.pow(5, deci);
-            }
-        } else if (deci < -27) {
-            d = value * Math.pow(5, -deci);
-
-        } else if (deci < 0) {
-            double fives = Maths.fives(-deci);
-            d = value * fives;
-
-        } else {
-            d = value;
-        }
-
-        double scalb = Math.scalb(d, exp - deci - scale2);
-        return negative ? -scalb : scalb;
+    public static double asDouble(long value, int exp, boolean negative, int deci) {
+        return Maths.asDouble(value, exp, negative, deci);
```

Comment:
Could we just delete this and delegate directly to the `Maths.asDouble(..)` instead? it was private previously so should be fine to remove.

Reply:
With a static import it should still be a minimal change